### ISSUE TITLE
SplitButton high contrast mode issues

### ISSUE THREAD

**Issue by khmakoto** [microsoft/fluentui#21411]
The a11y review of the v9 `SplitButton` component (#20917) identified a bunch of high contrast mode issues that need to be fixed besides the ones that are already pointed out in #21403 for the regular `Button` component: 

- The menu icon is not visible in non-primary appearance variants.

**Comment by khmakoto** [comment]
This is no longer an issue in the latest version.
